VogtPayroll5/PayrollConsoleReader.cs

```diff
@@ -24,5 +24,31 @@ public Employee GetEmployeeInfo()
 
             return emp;
         }
+
+        public bool ReturnTrueOrFalse()
+        {
+            string temp;
+            Console.WriteLine("Would you like to add another employee? y/n");
```

Comment:
Can simplify this into two lines to make it cleaner.

```c#
Console.WriteLine("Would you like to add another employee? y/n");
var temp = Console.ReadLine().ToLower();
```

Would have the same outcome.